roles/docker_image_build/vars/main.yml

```diff
@@ -94,3 +96,7 @@ role_run_commands:
   - command: "if [ -f {{ metricscheck_setup | quote }} ]; then python3 {{ metricscheck_setup | quote }} install; fi"
     user: root
     group: root
+
+role_docker_labels:
+  - name: org.opencontainers.image.base.name
+    value: "docker.io/library/ubuntu:{{ ubuntu_tag }}"
```

Comment:
Why is this content in both `vars/` and `defaults/`?

Reply:
It's not